src/wp-includes/script-loader.php

```diff
@@ -3409,7 +3409,7 @@ function wp_enqueue_block_style( $block_name, $args ) {
 }
 
 /**
- * Loads classic theme styles on classic themes in the frontend.
+ * Loads classic theme styles in themes which lack a theme.json file.
```

Comment:
Sorry, another tiny revision.

```suggestion
 * Loads classic theme styles when the current theme lacks a theme.json file.
```